recipe/meta.yaml

```
@@ -26,11 +26,14 @@ outputs:
         - wheel
       run:
         - python
-        - protobuf >=3.15.0,<5.0.0
-        - libprotobuf
+        - protobuf >=3.20.2,<6.0.0a0,!=3.20.0,!=3.20.1,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5
       run_constrained:
-        - grpcio >=1.0.0,<2.0.0
+        - grpcio >=1.44.0,<2.0.0
```

```suggestion
        - grpcio >=1.44.0,<2.0.0.dev0
```
See https://github.com/googleapis/python-api-common-protos/blob/3ef28bbd62f59a3b72c96b251e20481048b3d3e9/setup.py#L30C53-L30C58

---

addressed